dotnet/samples/Demos/OnnxSimpleRAG/Program.cs

```diff
@@ -3,15 +3,18 @@
 #pragma warning disable SKEXP0070
 #pragma warning disable SKEXP0050
 #pragma warning disable SKEXP0001
+#pragma warning disable SKEXP0020
 
 using System;
 using System.IO;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.VectorData;
 using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.InMemory;
+using Microsoft.SemanticKernel.Data;
 using Microsoft.SemanticKernel.Embeddings;
-using Microsoft.SemanticKernel.Memory;
-using Microsoft.SemanticKernel.Plugins.Memory;
+using Microsoft.SemanticKernel.PromptTemplates.Handlebars;
```

Comment:
Not very sure of adding this complexity to the demonstraction. Maybe having the `Simple` and a `HandlerBars` approach?

```suggestion
using Microsoft.SemanticKernel.PromptTemplates.Handlebars;
```

Reply:
Simple isn't working unfortunately.  The search response is returning a list of strings, but they are rendered as
`Answer the question using the memory content: System.Collections.Generic.List1[System.String]`

@markwallace-microsoft is adding a work item to try and get this addressed in the default prompt template engine